src/encoded/static/components/search.js

```diff
@@ -1451,7 +1452,7 @@ export class ResultTable extends React.Component {
         }
 
         return (
-            <div>
+            <React.Fragment>
```

Comment:
Since only one `<div>` is inside, could you just delete the previous `<div>` and replace it with nothing?